hs_composite_resource/models.py

```diff
@@ -447,16 +458,23 @@ def check_file_rename_or_move():
                     return aggregation.supports_resource_file_rename
                 except ObjectDoesNotExist:
                     # check if the source file represents an aggregation
-                    aggregation_path = os.path.join(aggregation_path, src_file_name)
-                    aggregation = self.get_aggregation_by_name(aggregation_path)
+                    # get source resource file object from source file path
+                    src_res_file = ResourceFile.get(self, src_file_name, aggregation_path)
+                    aggregation = src_res_file.logical_file
+                    if aggregation is None:
+                        raise ObjectDoesNotExist
```

Comment:
Should be consistent throughout the code to add an exception message, similar to lines 475-476 below

Reply:
Added exception message.